docs/extensions/services/authoring-database.md

```diff
@@ -0,0 +1,187 @@
+    await service.deleteOne(collection);
+    res.json();
+  });
+```
+
+## FieldsService
+
+The `FieldsService` provides access to perform CRUD operations on fields used in collections and item records.
+
+### Create a Field
+
+```js
+  router.post('/fields/:collection', async (req, res) => {
+    const schema = await getSchema();
+    const { collection } = req.params;
+
+    const service = new FieldsService({ schema });
+
+    const field = req.body;
+    await service.createField(collection, field);
+
+    const createdField = await service.readOne(collection, field.field);
+    res.locals['payload'] = { data: createdField || null };
+
+    res.json(createdField);
+  });
+```
+
+### Read Field
+
+```js
+ router.get('/fields/:collection', async (req, res) => {
+    const schema = await getSchema();
+
+    const { collection } = req.params;
+
+    const service = new FieldsService({ schema });
+
+    const data = await service.readAll(collection);
+    res.locals['payload'] = { data: data || null };
+    res.json(data);
+  });
+```
+
+### Update Field
+
+```js
+ router.patch('/fields/:collection/:field', async (req, res) => {
+    const schema = await getSchema();
+
+    const { collection, field } = req.params;
+
+    const service = new FieldsService({ schema });
+    await service.updateField(collection, { ...req.body, field });
+
+    const updatedField = await service.readOne(collection, field);
+    res.locals['payload'] = { data: updatedField || null };
+    res.json(updatedField);
+  });
+```
+
+Updating the field name is not supported at this time.
+
+### Delete Field
+
+```js
+ router.delete('/fields/:collection/:field', async (req, res) => {
+    const schema = await getSchema();
+    const { collection, field } = req.params;
+    const service = new FieldsService({ schema });
+    await service.deleteField(collection, field);
+    res.json();
+  });
+```
+
+## RelationsService
+
+The `RelationsService` allows you to manage relationships and references between items in Directus. Utilize it to create
+endpoints that perform CRUD operations on relations.
+
+### Create Relation
+
+```js
+router.post('/relations', async (req, res) => {
+    const schema = await getSchema();
+
+    const service = new RelationsService({ schema });
+
+    const data = await service.createOne(req.body);
+
+    const record = await service.readOne(data);
+    res.locals['payload'] = { data: record || null };
+
+    res.json(record);
+  });
+```
+
+### Get Relation
+
+```js
+router.get('/relations/:collection/:field', async (req, res) => {
+    const schema = await getSchema();
+
+    const service = new RelationsService({ schema });
+    const { collection, field } = req.params;
+
+    const data = await service.readOne(collection, field);
+
+    res.json(data);
+  });
+```
+
+::: tip Explore Services In-depth
+
+Check out the full list [here](https://github.com/directus/directus/blob/main/api/src/services).
+
+:::
```

Comment:
Please apply the same changes to this file as the ItemsService page: 

1. Rename based on nav suggestions
2. Add a generic 'this is how you initialize' example before the first method. in each section. 
3. Fix up indentation. 
4. End of file make sure link contains explainer.